src/particles/particle_accretion.hpp

```diff
@@ -23,12 +23,13 @@ namespace SinkAccretionUtils
 {
 
 constexpr int stencil_size = quokka::ParticleUtils::stencil_size;
+static constexpr double cs_floor_ = 1.33e4; // = 0.133 km/s = sqrt(k_B * 5 K / (2.33 m_p))
```

Comment:
I would prefer that we not hard wire in a separate floor here. We have a tempFloor already defined in hydro_system, and I think we should rely on that rather than setting a separate floor here. Just call hydro_system::enforceLimits before calling sink creation.